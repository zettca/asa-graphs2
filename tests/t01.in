8 11
8
3 5 -3
7 5 -2
6 1 1
5 6 -3
1 5 -2
5 4 -1
8 2 0
2 4 -1
3 2 3
8 7 -1
6 3 2
