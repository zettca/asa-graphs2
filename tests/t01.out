I
I
I
I
I
I
-1
0
